src/repository.py

```diff
@@ -248,7 +272,7 @@ def with_branch(self, branch_name: str) -> Iterator["Client"]:
         finally:
             self.switch(current_branch)
 
-    def get_summary(self, directory: str | None = DOCUMENTATION_FOLDER_NAME) -> DiffSummary:
+    def get_summary(self, directory: str | None = "") -> DiffSummary:
```

Comment:
Since we default `directory` at the GitHub level it can be dangerous to spread these kind of defaults throughout the repo since if they change in one place they have to change in all places. I'm guessing this is done to reduce the need to update tests? It would be good to remove the default and rely on the GitHub input default. Any thoughts on that please?

Reply:
Also applicable to any other places where we have defined the default

Reply:
Done!